northstar/src/runtime/island/args.rs

```diff
@@ -0,0 +1,38 @@
+// Copyright (c) 2021 ESRLabs
+//
+//   Licensed under the Apache License, Version 2.0 (the "License");
+//   you may not use this file except in compliance with the License.
+//   You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+//   Unless required by applicable law or agreed to in writing, software
+//   distributed under the License is distributed on an "AS IS" BASIS,
+//   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//   See the License for the specific language governing permissions and
+//   limitations under the License.
+
+use super::{ENV_NAME, ENV_VERSION};
+use std::ffi::CString;
+
+pub(super) fn args(
```

Comment:
You're right. I'll split that.

Regarding putting it to `Manifest`:
* `init`: A checked conversion to CString -> `pub fn cinit(&self) -> Result<CString, )>. See comment on you last comment regarding the use of `CString` vs `String` in `Manifest`
* `argv`: Looking again at the fn there's an error: the `.ok()` is not ok. Have to fix that. If we put this into `Manifest` then `argv` would return something different from what would expect. It would return an argv vector for the use with `exec*` and not just a getter to the `argv` field of the manifest. 
* `env`: I'd keep this separated from `Manifest` since it contains a runtime detail with `ENV_NAME` and `ENV_VERSION`. Mabe we should remove those two. What do you think?

Reply:
Added eaefa77